os/arch/Kconfig

```diff
@@ -735,34 +736,60 @@ config RAM_VSTART
 config RAM_SIZE
 	int "Primary RAM size"
 	default 0
+	depends on MM_NHEAPS = 1
 	---help---
 		The size in bytes of the installed primary RAM. "Primary" RAM
 		refers to the RAM that you link program code into.  If program code
 		does not execute out of RAM but from FLASH, then you may designate
 		any block of RAM as "primary."
+		This configuration can be used only MM_NHEAPS sets to 1.
 
 config RAMx_START
+	string "List of address for multiple heap memory"
+	default "0x00000000,"
+	depends on MM_NHEAPS != 1
+	---help---
+		The address list of the multiple heap memory.
+		When you want to use multi-heap in a RAM, set MM_NHEAPS first at memory
+		management, and then set hexa values at RAMx_START and RAMx_SIZE
+		to support multi-heap memory.
+		Each address is separated by ',' as shown in example.
+		i.e. "0x02000000,0x4000000"
+
+config RAMx_SIZE
```

Comment:
We had better change a name to clearly say the meaning of configs like PHI_RAMx_SIZE.